### DISCUSSION TITLE
esatus Wallet

### DISCUSSION THREAD

**Discussion by CHempel-esatus** [e-id-admin/public-sandbox-trustinfrastructure#17] [Wallets]
At this point, we would also like to thank Adnovum for setting up a self-service portal where anyone can create some basic identities. We have tested it with our esatus Wallet and it works perfectly. Try it yourself, the esatus Wallet is available here: 

App Store
[https://apps.apple.com/de/app/esatus-wallet/id1496769057](https://apps.apple.com/de/app/esatus-wallet/id1496769057)

Google Play Store
[https://play.google.com/store/apps/details?id=com.esatus.wallet](https://play.google.com/store/apps/details?id=com.esatus.wallet)

Adnovum Self-Service Portal: 
[https://ssi-start.adnovum.com/](https://ssi-start.adnovum.com/)

![esatus_Wallet](https://github.com/e-id-admin/public-sandbox-trustinfrastructure/assets/53933121/dc921ce1-46b3-4091-80d1-e0d3a45594fe)

**Comment by lanterno** [comment]
Hi @CHempel-esatus Thanks for making this available for us. I've just started working on EID sandbox, and I'm using esatus wallet, with our own issuer agent, and it's working well. 

However, I'm seeing a weird behaviour with using Adnovum. Once I receive the offer in the app, and try to view it, the app hangs, then shows "not responding". I tried a couple of times with the 3 Credential types available on Adnovum, and it's the same result. 

Do you have an idea what might be wrong here?
